DOCKER.md

```diff
@@ -0,0 +1,64 @@
+# Prerequisites
+Have docker-compose installed. Yes, it's that simple.  
+Don't have in installed yet, there's a detailed explanation on their
+[website](https://docs.docker.com/compose/install/).
+
+# How to use
+* clone the repository `git clone git@github.com:jasperweyne/helpless-kiwi.git`
+* navigate to the cloned repository
+* run `docker-compose up -d` the first time this will take some time because it
+  needs to build everything. Disregard any red text you might see, this is just
+  part of setting up the containers.
+* run `docker ps` to see if all 3 services listed below should be up.
+	* kiwi_symfony-dev
+	* kiwi_phpmyadmin-dev
+	* kiwi_database-dev
+* assuming they are, run `composer db-rebuild-dev`
```

Comment:
This should be done from inside the container, that should be explicitly documented

Reply:
my bad, completely read over this. thanks for catching it, kind of an important distinction.